tests/specs/geoQuery.spec.js

```diff
@@ -1376,5 +1376,29 @@ describe("GeoQuery Tests:", function() {
         cl.x("p5");
       }).catch(failTestOnCaughtError);
     });
+
+    it("Calling cancel() in the middle of firing \"key_entered\" events is allowed", function(done) {
+      var cl = new Checklist(["p1", "cancel query", "loc1 entered"], expect, done);
```

Comment:
Order doesn't matter for the `Checklist`, but I'll change the order for clarity.